src/Tribe/Commerce/PayPal/Custom_Argument.php

```diff
@@ -57,6 +57,20 @@ public static function decode( $encoded, $assoc_array = false ) {
 			$encoded = str_replace( '\"', '"', $encoded );
 		}
 
+		// in case we receive the param in non json 'user_id:0,tribe_handler:tpp,pid:4' format
+		if ( ! strpos( $encoded, '{' ) && ! strpos( $encoded, '}' ) ) {
```

Comment:
You might need to check here exactly for false instead of a falsy value as 0 might be returned from `strpos` and is false